.travis.yml

```diff
@@ -1,13 +1,24 @@
-language: node_js
-node_js: "node"
-
 # OS
 dist: xenial
 
-# install dependencies
-install:
-  - npm install
+matrix:
```

Comment:
Hmmm, not super happy about having to put this into a matrix when there is only one test run. But otherwise, it was going to force me to put all the environment variables on the same line, which was going to be ugly and would make it difficult to keep track of the encrypted variables.